### ISSUE TITLE
BUG | Video Playlist using Vimeo doesn't properly stop iphone from auto locking during playback

### ISSUE THREAD

**Issue by Joeyphats** [elementor/elementor#16234]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Playing a video from Vimeo via the playlist widget doesn't properly stop the iPhone auto lock feature from engaging. Which means that screen will turn off and stop playback. It happens with Safari or Chrome.

## Steps to reproduce

Add a new video playlist widget. 
Add a video with Vimeo as the source. Add a second video from self-hosted to compare with (optional). 
Play the video on an iPhone in full screen the expected behavior is that the video will play through completion, however if your screen timeout is shorter than the video length it will lock during playback.  (I was testing with an iPhone 12 with iOS 14.7.1. A user on our site has the same issue with an iPhone 11 on 14.4)

This does not happen using self-hosted as the video source. 
This behavior also doesn't seem to happen on iPadOS or Android. I have a sample page here: https://www.fabfitbylw.com/temp-elementor-support-a/

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
[system-info-www.fabfitbylw.com-01-09-2021.txt](https://github.com/elementor/elementor/files/7094884/system-info-www.fabfitbylw.com-01-09-2021.txt)

</details>

**Comment by Joeyphats** [comment]
I did get confirmation via a support ticket that one of your L2 techs was able to successfully duplicate the issue. They did close my ticket since I had opened up this bug report.  This issue is impacting my production site (our limited test users didn't find this use case). We've been able to work around it and rolling back removes the Vimeo feature from the playlist widget which we don't want. So we're going with fix in place here. Just wondering if a dev has has a chance to peek at this issue.  Thanks,
Joe
